testsuite/fuzzer/fuzz/fuzz_targets/move/aptosvm_publish_and_run.rs

```diff
@@ -51,21 +51,28 @@ static TP: Lazy<Arc<rayon::ThreadPool>> = Lazy::new(|| {
 
 const MAX_TYPE_PARAMETER_VALUE: u16 = 64 / 4 * 16; // third_party/move/move-bytecode-verifier/src/signature_v2.rs#L1306-L1312
 
-const EXECUTION_TIME_GAS_RATIO: u8 = 50;
+const EXECUTION_TIME_GAS_RATIO: u8 = 100;
+
+// List of known false positive messages for invariant violations
+// If some invariant violation do not come with a message, we need to attach a message to it at throwing site.
+const KNOWN_FALSE_POSITIVES: &[&str] = &["too many type parameters/arguments in the program"];
```

Comment:
I guess we do not expect this set to be large so iterating over a linear sequence is ok?

Reply:
Indeed, a short list is anticipated, and we only iterate when the fuzzer triggers an invariant.